tests/app/main/views/test_services.py

```diff
@@ -6,32 +6,86 @@
     from io import BytesIO as StringIO
 import mock
 
+from lxml import html
+
 from dmapiclient import HTTPError, REQUEST_ERROR_MESSAGE
 from ...helpers import LoggedInApplicationTest
 
 
 class TestServiceView(LoggedInApplicationTest):
     @mock.patch('app.main.views.services.data_api_client')
-    def test_service_response(self, data_api_client):
+    def test_service_view_no_features_or_benefits_status_disabled(self, data_api_client):
         data_api_client.get_service.return_value = {'services': {
-            'frameworkSlug': 'g-cloud-8'
+            'frameworkSlug': 'g-cloud-8',
+            'id': "314159265",
+            "status": "disabled",
         }}
-        response = self.client.get('/admin/services/1')
+        response = self.client.get('/admin/services/314159265')
 
-        data_api_client.get_service.assert_called_with('1')
+        assert data_api_client.get_service.call_args_list == [
```

Comment:
This feels less nice than `assert_called_with`, especially with the not-very-obvious empty dict in there. What's the motivation behind changing how we test the call here?